Tools/smc-fuzzer/smc.cpp

```diff
@@ -54,7 +54,7 @@ float fpe2ToFlt(char *str, int size) {
   if (size != 2)
     return 0;
 
-  uint16_t value = ((uint16_t)str[0] << 8) | str[1];
+  uint16_t value = ((uint16_t)str[0] << 8) + str[1];
```

Comment:
Could you try with (uint16_t) str[1] instead? Also, this is third-party code, we recommend our own smcread tool.

Reply:
Will try and let you know

Reply:
Ping?